api_cgn.yaml

```diff
@@ -171,6 +171,43 @@ paths:
           schema:
             $ref: "#/definitions/ProblemJson"
   
+  "/delete":
+    post:
```

Comment:
just out of my curiosity, why not define new api as `/activation` with `delete` method?

Reply:
Because we let this separated from activation procedure. We must keep separated in case we have to get unsubscription procedure status (that differs from activation procedure)